start.sh

```diff
@@ -111,20 +180,18 @@ echo "Outputting logs now:"
 docker-compose logs -f &
 PROCESS=$!
 
+#watchdog
 while [ "$STOPPING" != 'true' ]; do
```

Comment:
This is a workaround for a MAC issue: a system sleep is not communicated to the `docker VM` this screws up the clock in the docker containers, making it dead slow, does this also exits in windows? probably no need to monitor the container clock in windows.

Reply:
I need a bit more info - let's do a call.